src/com/cft/contactmerge/contact/Address.java

```diff
@@ -0,0 +1,96 @@
+package com.cft.contactmerge.contact;
+
+public class Address {
+    String streetAddress;
+    String apartment;
+    String city;
+    String state;
+    String country;
+    String zip;
+    String addressType;
+
+    /*******************************************************************************************************************
+     *************************************************** Constructors **************************************************
+     *******************************************************************************************************************/
+    public Address(){
+        setStreetAddress("");
+        setApartment("");
+        setCity("");
+        setState("");
+        setCountry("");
+        setZip("");
+        setAddressType("");
+    }
+
+    public Address(String streetAddress, String apartment, String city, String state, String country, String zip, String addressType){
+        setStreetAddress(streetAddress);
+        setApartment(apartment);
+        setCity(city);
+        setState(state);
+        setCountry(country);
+        setZip(zip);
+        setAddressType(addressType);
+    }
+
+    /*******************************************************************************************************************
+     *************************************************** Get Methods ***************************************************
+     *******************************************************************************************************************/
+    public String getStreetAddress() {
+        return this.streetAddress;
+    }
+
+    public String getApartment() {
+        return this.apartment;
+    }
+
+    public String getCity() {
+        return this.city;
+    }
+
+    public String getState() {
+        return this.state;
+    }
+
+    public String getCountry(){
+        return this.country;
+    }
+
+    public String getZip(){
+        return this.zip;
+    }
+
+    public String getAddressType() {
+        return this.addressType;
+    }
+
+    /*******************************************************************************************************************
+     *************************************************** Set Methods ***************************************************
+     *******************************************************************************************************************/
+    public void setStreetAddress(String streetAddress) {
```

Comment:
My personal preference is to keep sets/gets together as it is important to make sure the behavior is consistent and understandable between the getter and setter. If you prefer them apart we should add this to our "standards" and start doing all the others this way.

Reply:
Will do, and change the comment to Get/Set Methods.